scripts/typesBundle.mts

```diff
@@ -0,0 +1,131 @@
+export default {
```

Comment:
It was already done by @youPickItUp before I'd taken over the ticket, but it must be coming from [the `apps` (azero.dev) code](https://github.com/Cardinal-Cryptography/apps/blob/ac67ebf44af69f32142c8096b783d7db7d8f170b/packages/apps-config/src/api/spec/aleph-zero.ts).

Reply:
Let me actually format it so it looks like js instead of json (not sure why eslint has not caught that, but eslint in this repo needs a revamp overal so I'm not even spending time on looking into it).